code/game/machinery/camera/camera_assembly.dm

```diff
@@ -121,17 +121,24 @@
 	else if(I == proxy_module)
 		proxy_module = null
 
+/obj/structure/camera_assembly/welder_act(mob/living/user, obj/item/tool)
+	if(state != STATE_WRENCHED && state != STATE_WELDED)
+		return
+	. = TRUE
+	if(!tool.tool_start_check(user, amount=3))
+		return
+	user.balloon_alert_to_viewers("started [state == STATE_WELDED ? "un" : null]welding [src]")
```

Comment:
```suggestion
	user.balloon_alert_to_viewers("[state == STATE_WELDED ? "un" : null]welding...")
```

`doing thing...` for in progress actions, and avoid including "src", since it makes the alerts too long